deployment/docker-compose.yml

```diff
@@ -109,6 +109,7 @@ services:
       GATEWAY_SERVER_URL: http://aerie_gateway:9000
       HASURA_CLIENT_URL: http://localhost:8080/v1/graphql
       HASURA_SERVER_URL: http://hasura:8080/v1/graphql
+      ORIGIN: http://localhost
```

Comment:
Is the expectation that users would edit this value to match their url?

Reply:
Yes exactly. So for example if they deploy the UI server behind an HTTPS proxy you can update the ORIGIN to be https.